### PR TITLE
Update zhtlc docs

### PR DESCRIPTION
**Pull request by smk762**
ref: https://github.com/KomodoPlatform/komodo-defi-framework/pull/1922

### CODE REVIEW

File: src/pages/atomicdex/api/v20/index.mdx [hunk 1]
```diff
@@ -106,9 +106,155 @@ It includes a uniform request, successful and error response formats. At the mom
...
+| zcash_params_path         | string    | ZHTLC coins only. Path to folder containing [Zcash parameters](https://z.cash/technology/paramgen/). Optional, defaults to standard location as defined in [this guide](https://forum.komodoplatform.com/t/installing-zcash-params/603)                |
+| scan_blocks_per_iteration | integer   | ZHTLC coins only. Sets the number of scanned blocks per iteration during `BuildingWalletDb` state. Optional, default value is 1000.                                                                                                                    |
+| scan_interval_ms          | integer   | ZHTLC coins only. Sets the interval in milliseconds between iterations of `BuildingWalletDb` state. Optional, default value is 0.                                                                                                                      |
+| sync_params               | integer or string | ZHTLC coins only. Optional, defaults to two days ago. Defines where to start scanning blockchain data upon initial activation. Options: `"earliest"` (the coin's sapling_activation_height), `height` (a specific block height) or `date` (a linux timestamp).            |
```
Comment (borngraced) [inline src/pages/atomicdex/api/v20/index.mdx:130]: I think the term `a unix timestamp` sounds cooler than linux.

**Review by borngraced — APPROVED** [review]
🔥 Just a comment

File: src/pages/atomicdex/api/v20/index.mdx (hunk 1 above)
Comment (smk762) [reply src/pages/atomicdex/api/v20/index.mdx]: agree - done

File: src/pages/atomicdex/api/v20-dev/zhtlc_coins/index.mdx [hunk 2]
```diff
@@ -85,26 +65,116 @@ To withdraw ZHTLC coins, you need to use the [task::withdraw](/atomicdex/api/v20
...
 
-  ```json
-  {
-    "mmrpc": "2.0",
-    "result": {
-      "task_id": 0
-    },
-    "id": null
+#### Sync from sapling activation height (earliest)
+
```
Comment (borngraced) [inline src/pages/atomicdex/api/v20-dev/zhtlc_coins/index.mdx:101]: we should explain why sapling activation height is the earliest.

Reason: `ZCoin lightwallet client supports only blocks that are post sapling.`

**Review by borngraced — CHANGES_REQUESTED** [review]
last note.

File: src/pages/atomicdex/api/v20-dev/zhtlc_coins/index.mdx (hunk 2 above)
Comment (smk762) [reply src/pages/atomicdex/api/v20-dev/zhtlc_coins/index.mdx:101]: updated, thanks

**Review by borngraced — APPROVED** [review]
lgtm 🚀

**Review by gcharang — APPROVED** [review]

**Pull request merged**